script/MintAndListRemovals.s.sol

```diff
@@ -0,0 +1,47 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity =0.8.15;
+
+import "forge-std/Script.sol";
+import "../contracts/Removal.sol";
+import {UInt256ArrayLib} from "../contracts/ArrayLib.sol";
+
+contract MintAndListRemovals is Script {
```

Comment:
why can't we just use [`cast send ...`](https://book.getfoundry.sh/reference/cast/cast-send)  for stuff like this?